docs/api/java/quickstart.rst

```diff
@@ -0,0 +1,176 @@
+Quickstart Guide
+================
+
+First, create a cvc5 `Solver <io/github/cvc5/api/Solver.html>`_ instance using try with resources:
+
+.. code-block:: java
+
+     try (Solver solver = new Solver())
+     {
+       /** write your code here */
+     }
+
+We will ask the solver to produce models and unsat cores in the following,
+and for this we have to enable the following options.
```

Comment:
```suggestion
To produce models and unsat cores, we have to enable the following options.
```

(I know that this is just copied from the other guides, but the wording is a bit verbose)